src/checkUsageCalc.ts

```diff
@@ -0,0 +1,63 @@
+import { CheckType } from './types';
+
+interface ActiveSeriesParams {
+  probeCount: number;
+  checkType: CheckType;
+  frequencySeconds: number;
+}
+
+interface UsageValues {
+  checksPerMonth: number;
+  activeSeries: number;
+  logsGbPerMonth: number;
+}
+
+enum CheckSeries {
+  PING = 76,
+  HTTP = 128,
+  DNS = 79,
+  TCP = 59,
+}
+
+const getSeriesPerCheck = (checkType: CheckType) => {
+  switch (checkType) {
+    case CheckType.PING:
+      return CheckSeries.PING;
+    case CheckType.TCP:
+      return CheckSeries.TCP;
+    case CheckType.DNS:
+      return CheckSeries.DNS;
+    case CheckType.HTTP:
+      return CheckSeries.HTTP;
+  }
+};
+
+const getMonthlyChecks = (frequencySeconds: number) => {
```

Comment:
**Nitpick:** I would probably call it `getChecksPerMonth()` instead as we are referring like that to its return value everywhere else.